tests/kernel/wave/speculative_decode_test.py

```diff
@@ -37,6 +39,7 @@ def get_wave_speculative_decoding_kernel(shape: int):
         waves_per_eu=2,
         denorm_fp_math_f32="preserve-sign",
         schedule=False,
+        wave_runtime=True,
```

Comment:
Just curious, why was this needed?

Reply:
scalar arguments are currently only available/working with wave_runtime